plugins/generator-1.16.5/forge-1.16.5/procedures/get_command_parameter_entities_foreach.java.ftl

```diff
@@ -0,0 +1,7 @@
+try {
+	EntityArgument.getEntitiesAllowingNone(arguments, "${field$param}").forEach(entityiterator -> {

```

Comment:
Use standard `for (element : list)` iteration here as it allows for more relaxed variable access. Same for 1.18.2